components/Column.tsx

```diff
@@ -25,41 +47,28 @@ const Column = ({ idx, state, stories, addFilter }: ColumnParams): JSX.Element =
   const { palette } = useTheme();
 
   return (
-    <Col
+    <ColumnContainer
       key={idx}
-      style={{
-        borderTop: `5px solid ${colors[state]}`,
-        backgroundColor: palette.accents_2,
-        boxShadow: `1px 1px 5px 0px ${palette.accents_1}`,
-        margin: 10,
-        padding: 5,
-        height: '100%',
-      }}
+      bordertop={`5px solid ${colors[state]}`}
```

Comment:
I think i'd prefer to have the colors as props, rather then the names of the styles, wdyt?